tests/integration/test_jemalloc_percpu_arena/test.py

```diff
@@ -48,15 +48,20 @@ def skip_if_jemalloc_disabled():
         SELECT value FROM system.build_options WHERE name = 'USE_JEMALLOC'"
     """).strip()
     if output != b'ON' and output != b'1':
-        pytest.skip(f'Compiled w/o jemalloc (USE_JEMALLOC={output})')
+        return True
 
+    return False
 
 # Ensure that clickhouse works even when number of online CPUs
 # (_SC_NPROCESSORS_ONLN) is smaller then available (_SC_NPROCESSORS_CONF).
 #
 # Refs: https://github.com/jemalloc/jemalloc/pull/2181
 def test_jemalloc_percpu_arena():
-    skip_if_jemalloc_disabled()
+    # TODO: revert me when https://github.com/ClickHouse/ClickHouse/pull/33162
```

Comment:
This is not the only test that uses `pytest.skip`, and any test should be converted to this then, but I think it is better to revert this hunk, since this is the problem if there is some other in the test group had been failed, and for this particular test it was `test_input_format_parallel_parsing_memory_tracking`, that had been fixed in #33163